References.php

```diff
@@ -184,7 +184,7 @@ public function rotation(string $rotation): string
         ));
     }
 
-    public function blur(string $blur): float
+    public function blur(string $blur): string
     {
         $blur = \trim($blur);
 
```

Comment:
Since the function returns `(float) ...`, you also need to remove the type-cast on the line below this one  😄

Reply:
O_o
Because of my old php version, I'm using the 2.0.2 version and I had to change related version😄
I'll do it soon.

Reply:
I didn't find any changelog,
Would you like to create one?
It will make maintainability easier and we can fix any problems in older version which have been used by older php versions.

Reply:
I will add a changelog file for next release, I'll include your changes in it 😉

Reply:
Very well:-)

Reply:
@Pierstoval It seems that the master branch does not yet integrate the correction. Is it normal?

Reply:
It seems it's resolved
@Pierstoval  you could add my name as a contributor!

Reply:
@AMK9978 Done: https://github.com/Orbitale/ImageMagickPHP/blob/main/CHANGELOG.md#fixes-1